### PR TITLE
Feat: add press release "Four Northern California..."

### LINKED ISSUE #245: New press request
### Title

Four Northern California transit agencies join forces to buy contactless open-loop fare payment systems off of California’s purchasing agreements 

### Heading (optional)

_No response_

### Asset URL

https://docs.google.com/document/d/1eUIWkXAgUCcFKR9MqIHA9hMtqulA3bb7R8bhp6e5Eps/edit

### Tag

Contactless payments

### Date - Year

2023

### Date - Month

08

### Location

_No response_

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #245

### CODE REVIEW

File: src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+tags:
+  - Contactless Payments
+outlet: "Cal-ITP"
+---
+
+Humboldt Transit Authority, Lake Transit Authority, Mendocino Transit Authority, and Redwood Coast Transit Authority—collectively part of the Far North Group—are the first group of agencies to band together to use the State of California’s competitively priced contracts to purchase and install the hardware and software services needed for accepting customers’ debit and credit cards and mobile wallets for fare payments.
+
+Cal-ITP—Caltrans’ California Integrated Travel Project—supported the agencies through the process of purchasing open-loop payment acceptance devices and fare calculation software from the State of California’s competitively awarded [Master Service Agreements](https://www.calitp.org/assets/Contactless.Payments.MSA.pdf?utm_source=Press+Release&utm_medium=Website&utm_id=FNG) (MSAs), contracts that allow U.S. transit agencies to purchase directly from vendors without further competitive bidding.
```
Comment (angela-tran) [inline src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md:16]: Most of the links in the asset document contained this stuff at the end: `?utm_source=Press+Release&utm_medium=Website&utm_id=FNG`. Is that actually desired, or should I remove those?

File: src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+“Paying for travel should be as simple as buying a cup of coffee, and adding contactless debit and credit card readers makes transportation more accessible and user-friendly for all riders. It also facilitates the seamless connections that travelers want and need,” says Gillian Gillett, Caltrans’ Data & Digital Services Division Chief and California Integrated Mobility Program Manager. Gillett leads [Cal-ITP](https://www.calitp.org/?utm_source=Press+Release&utm_medium=Website&utm_id=FNG), a California-based initiative to make riding by bus and rail simpler and more cost-effective—for public transportation providers and riders—by standardizing trip-planning information and modernizing fare collection systems.
```
Comment (angela-tran) [inline src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md:22]: Notice the `?utm_source=Press+Release&utm_medium=Website&utm_id=FNG` part in the URL

File: src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## About Cal-ITP
+The California Integrated Travel Project (Cal-ITP) was established by the California State Transportation Agency (CalSTA) and California Department of Transportation (Caltrans) to both improve and encourage the use of multimodal travel throughout California—by enabling contactless open-loop payments, standardizing information for easy multimodal trip planning, and automating customer discounts ([Cal-ITP Benefits](https://www.calitp.org/assets/Cal-ITP.Benefits.PressRelease.220921.pdf?utm_source=Press+Release&utm_medium=Website&utm_id=FNG)). Over the past two years, Cal-ITP successfully led contactless open-loop contactless payment implementations in California on Monterey-Salinas and Santa Barbara buses; Sacramento light rail; intercity passenger rail between Sacramento and the San Francisco Bay Area; on-demand transit vans; and LAX’s FlyAway bus, which connects airport passengers to commuter rail.
```
Comment (angela-tran) [inline src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md:32]: Notice the `?utm_source=Press+Release&utm_medium=Website&utm_id=FNG` part in the URL

File: src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Learn more at [calitp.org](https://www.calitp.org/?utm_source=Press+Release&utm_medium=Website&utm_id=FNG) and on [@California_ITP](https://twitter.com/california_itp?utm_source=Press+Release&utm_medium=Website&utm_id=FNG) on Twitter, or by subscribing to the [Caltrans Mobility Newsletter](https://lp.constantcontactpages.com/su/eLbtFoE/calitp?utm_source=Press+Release&utm_medium=Website&utm_id=FNG). And visit Cal-ITP’s [CAMobilityMarketplace.org](https://www.camobilitymarketplace.org/?utm_source=Press+Release&utm_medium=Website&utm_id=FNG) for a catalog of code-compliant products and services for public mobility providers, including contactless payment acceptance hardware and software and competitively priced cellular data plans.
```
Comment (angela-tran) [inline src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md:34]: Notice the `?utm_source=Press+Release&utm_medium=Website&utm_id=FNG` part in the URLs

File: src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md (hunk 1 above)
Comment (angela-tran) [reply src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md:16]: @esquared415 @mrose914 - See question above

**Comment by angela-tran** [comment]
Fixed an issue where the post's datetime was in the future, so it wasn't showing up

**Comment by esquared415** [comment]
> 

@angela-tran I got an alert on this and wasn't sure what you meant here (today is 8/18/23 ... did I goof a date somewhere? thanks for helping me learn!).

**Comment by angela-tran** [comment]
> > 
> 
> @angela-tran I got an alert on this and wasn't sure what you meant here (today is 8/18/23 ... did I goof a date somewhere? thanks for helping me learn!).

@esquared415 Oh, no the date in your source document is fine. The incorrect datetime was entirely on my end and [has been fixed](https://github.com/cal-itp/calitp.org/pull/246/commits/0c7a9326b8a2bd16cbcd16b759370b7f5dc36583) - I wrote that comment above just for my future self and other developers working on the site.

Here's a direct link to the press release on the preview site: https://deploy-preview-246--cal-itp-website.netlify.app/press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements

File: src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md (hunk 1 above)
Comment (angela-tran) [reply src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md:16]: Got it! Thanks

**Review by thekaveman — APPROVED** [review]
Looks good to me! Let's wait until @esquared415 approves the preview.

**Homepage**

image

**Press Page**

_With the filter pill selected_

image

**Review by esquared415 — APPROVED** [review]
@angela-tran @thekaveman This looks sooo good as a webpage instead of a standalone PDF—I love it! Real website content!! 📰 🥳 

One Q for @mrose914 (but nothing that should hold us up from publishing): While I can see "Aug 2023" under the headline on the "Press" page, I don't see a date field on the press release itself—so if someone shares the direct link to the press release (bypassing the Press page), they'd have no sense of how recently this was published.

I think it would be worth having that month / year field appear somewhere on the press releases as well.

Thanks for considering, and please pub awayyyy!

**Pull request merged**